docs/team/weijie96.md

```diff
@@ -12,26 +12,45 @@ It is written in Java, and has about 10 kLoC.
 Given below are my contributions to the project.
 
 * **New Feature**: Added the ability to archive clients.
-  * What it does: **Coming Soon**
-  * Justification: **Coming Soon**
-  * Highlights: **Coming Soon**
+  * What it does: It allows the user to add active clients into the archive. The opposite can be done as well; archived clients can be made active again.
```

Comment:
Use the feature instead of it. (Just minor thing)

Reply:
Done